scripts/pcvGuardianDeploy.ts

```diff
@@ -0,0 +1,57 @@
+import hre, { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { utils } from 'ethers';
+import { keccak256 } from 'ethers/lib/utils';
+import config from './config';
+import { Core } from '@custom-types/contracts';
+
+const { CORE, PROTOCOL_MULTISIG_ADDRESS, VOLT_FUSE_PCV_DEPOSIT, PCV_DEPOSIT } = config;
+
+const PCV_GUARD_ROLE = keccak256(utils.toUtf8Bytes('PCV_GUARD_ROLE'));
+
+const pcvGuardAddress1 = '0xf8D0387538E8e03F3B4394dA89f221D7565a28Ee';
+const pcvGuardAddress2 = '0xd90E9181B20D8D1B5034d9f5737804Da182039F6';
+
+async function deploy() {
+  const PCVGuardian = await ethers.getContractFactory('PCVGuardian');
+  const whitelistAddresses = [VOLT_FUSE_PCV_DEPOSIT, PCV_DEPOSIT];
```

Comment:
We should also add the new PSM to this list of addresses that can be pulled from